config/style.yml

```diff
@@ -13,14 +13,6 @@ Style/IfUnlessModifier:
 Style/MethodCalledOnDoEndBlock:
   Enabled: true
 
-# Part of the orignal GDS styleguide
-# "Use Ruby 1.9 syntax for symbolic hash keys.
-# This includes method calls."
-# https://github.com/alphagov/styleguides/blob/6395a10d41c3938f4c147cda443fd83f854c3e7a/ruby.md#collections
-Style/HashSyntax:
-  Exclude:
-  - 'db/schema.rb'
```

Comment:
(We also [ignore this file in general](https://github.com/alphagov/rubocop-govuk/blob/master/config/default.yml#L7).)